package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,777 @@
+          10. "angle"
+
+      It takes up more space than :attr:`~WaterBridgeAnalysis.timeseries` but
+      it is easier to analyze and to import into databases or dataframes.
+
+
+      .. rubric:: Example
+
+      For example, to create a `pandas.DataFrame`_ from ``h.table``::
+
+         import pandas as pd
+         df = pd.DataFrame.from_records(w.table)
+"""
+import numpy as np
+import logging
+import warnings
+
+from .hbond_analysis import HydrogenBondAnalysis
+from MDAnalysis.lib.NeighborSearch import AtomNeighborSearch
+from MDAnalysis.lib.log import ProgressMeter, _set_verbose
+from MDAnalysis import SelectionWarning
+
+logger = logging.getLogger('MDAnalysis.analysis.wbridges')
+
+class WaterBridgeAnalysis(HydrogenBondAnalysis):
+    """Perform a water bridge analysis
+
+    The analysis of the trajectory is performed with the
+    :meth:`WaterBridgeAnalysis.run` method. The result is stored in
+    :attr:`WaterBridgeAnalysis.timeseries`. See
+    :meth:`~WaterBridgeAnalysis.run` for the format.
+
+    :class:`WaterBridgeAnalysis` uses the same default atom names as \ :class:`~MDAnalysis.analysis.hbonds.hbond_analysis.HydrogenBondAnalysis`, see :ref:`Default atom names for hydrogen bonding analysis`
+
+
+    .. versionadded:: 0.17.0
+    """
+    def __init__(self, universe, selection1='protein', selection2='not resname SOL', water_selection='resname SOL',
+                 selection1_type='both', update_selection1=False, update_selection2=False, update_water_selection = True,
+                 filter_first=True,
+                 distance_type='hydrogen', distance=3.0, angle=120.0, forcefield='CHARMM27', donors=None,
+                 acceptors=None, start=None, stop=None, step=None, debug=None,
+                 verbose=None):
+        """Set up calculation of water bridges between two selections in a universe.
+
+        The timeseries is accessible as the attribute :attr:`WaterBridgeAnalysis.timeseries`.
+
+        If no hydrogen bonds are detected or if the initial check fails, look
+        at the log output (enable with :func:`MDAnalysis.start_logging` and set
+        `verbose` ``=True``). It is likely that the default names for donors
+        and acceptors are not suitable (especially for non-standard
+        ligands). In this case, either change the `forcefield` or use
+        customized `donors` and/or `acceptors`.
+
+        Parameters
+        ----------
+        universe : Universe
+            Universe object
+        selection1 : str (optional)
+            Selection string for first selection ['protein']
+        selection2 : str (optional)
+            Selection string for second selection ['not resname SOL']
+            This string selects everything except water where water is assumed
+            to have a residue name as SOL.
+        water_selection : str (optional)
+            Selection string for bridging water selection ['resname SOL']
+            It is assumed that the water has a residue name SOL.
```

Comment:
Replace this line with "The default selection assumes that the water molecules have residue name "SOL". Change it to the appropriate selection for your specific force field."

Reply:
@orbeckst Thank you. I have changed it.